src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,48 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.bp-ColorPickerPalette {
+    position: absolute;
+    top: -55px;
+    right: 50%;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: fade-out($black, .2);
+    border-radius: 4px;
+    transform: translateX(50%);
+
+    &::after {
+        position: absolute;
+        top: 75%;
+        width: 18px;
+        height: 18px;
+        background: inherit;
+        border-radius: 0 0 0 6px;
+        transform: rotate(-45deg);
+        content: '';
+        clip-path: polygon(0% 0%, 100% 100%, 0% 100%);
+    }
+}
+
+.bp-ColorPickerPalette-button {
+    width: 20px;
+    height: 20px;
+    margin: 10px 2px 10px 2px;
+    padding: 0;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+
+    &:first-child {
+        margin-left: 9px;
+    }
+
+    &:hover {
+        box-shadow: 0 0 0 2px white;
```

Comment:
We should try to avoid named CSS colors. Can we use `$white` or `#fff`, instead?